services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HakemusBuilder.kt

```diff
@@ -210,6 +210,17 @@ data class HakemusBuilder(
                     .tyonSuorittaja(founder())
         }
 
+    /**
+     * Make the hakemus appear like it has been just sent. I.e. it has mandatory fields and allu
+     * status fields filled.
+     */
+    fun asSent(
+        status: ApplicationStatus? = ApplicationStatus.PENDING,
+        alluId: Int? = 1,
+        identifier: String? = "JS000$alluId",
```

Comment:
Maybe it would be better to use a type-specific identifier? E.g. `JS2400001` or `KP2400001`.

Reply:
I made the identifier depend on the application type.